todo-app/src/App.js

```diff
@@ -33,10 +36,13 @@ export default class App extends Component {
 			<h4 className='task-done'>✓ = {tasksStore.doneCount}</h4>
 			<h3>Enter description:</h3>
 			<TaskForm tasksStore = {tasksStore} />
-	  	{tasksStore.tasks.length === 0 ? 
-	  		<div>No items...</div> 
-	  	: 
-				<ul className='task-list'>
+	  	{tasksStore.loadingTasks ? (
```

Comment:
When you have nested ternaries like this it starts to get pretty hard to read. Can you refactor this into multiple returns with `if` statements so that it's more clear?